lib/project_types/script/layers/application/supported_languages.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Script
+  module Layers
+    module Application
+      class SupportedLanguages
+        STABLE_LANGUAGES = %w(AssemblyScript)
```

Comment:
To follow this naming convention we'd probably need to be certain that AS is stable, and it's but I think that statement is subjective 😄 To disambiguate this a bit more, what do you think of?

```
class Languages
  SUPPORTED_LANGUAGES = %(...)
  BETA_LANGUAGES = %(...)
  ...
end
```

Reply:
I actually ended up removing this class and now we check what sdks are defined per EP to know what languages are available/beta. I think it makes more sense because this way the EP defines what languages are supported. So in the EP config there is something like:

```
payment_filter:
  assemblyscript:
    package: "@shopify/extension-point-as-payment-filter"
    sdk-version: "^7.0.0"
    toolchain-version: "^3.0.0"
  rust:
    beta: true
    package: "@shopify/extension-point-rs-payment-filter"
```